proxy/http/HttpConnectionCount.cc

```diff
@@ -180,7 +197,7 @@ OutboundConnTrack::obtain(TxnConfig const &txn_cnf, std::string_view fqdn, IpEnd
   if (loc != _imp._table.end()) {
     zret._g = loc;
   } else {
-    zret._g = new Group(key, fqdn);
+    zret._g = new Group(key, fqdn, txn_cnf.min);
```

Comment:
Why does `min` need to be set here, but not `max`?

Reply:
Not really - `max` just as overridable as `min` would be after this PR. The reason for not updating _max_ into `Group` is that, the configured _max_ value only needed when making a new Origin connection (to check if the current count is past the _max_), and we always have  a valid TXN when making a new origin connection.

However, that's not true for `min` - `min` value is needed when closing an existing _idle_ (keep alive) connection, which generally happens because of a VC level event (an error, or a keep alive timeout etc) and it isn't a guaranteed to have a valid `TXN` object (and therefore it's config) to compare against :( 

If it makes sense to keep it consistent and remove confusion, I'm okay to add `max` also to the `Group`, but, it'd be redundant.